packages/styles/tag-button.css

```diff
@@ -0,0 +1,8 @@
+.TagButton {
+  vertical-align: middle;
+}
```

Comment:
I'm not sure this is needed.

Reply:
I have added this, because of <Tag> component. Button has this style, when we add Tag and Button in a row, it won't be on the one level height, but since we need to remove Tag component (as i understood from above messages) it won't be conflicted